app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -23,6 +23,24 @@ def permission_check(verb, klass = controller_name.classify.constantize)
           access_obj = RBAC::Access.new(klass.table_name, verb).process
           raise Catalog::NotAuthorized, "#{verb.titleize} access not authorized for #{klass}" unless access_obj.accessible?
         end
+
+        def permission_array_check(verbs)
+          return unless RBAC::Access.enabled?
+
+          if verbs.kind_of?(Array)
```

Comment:
Possibly I'm mis reading this. But the conditional is checking if `verbs` is an `Array` but then erroring out saying it must be an array. Same with the next line checking if they are all `String` and then erroring saying everything should be a string.

Reply:
Yep had them backwards. So thats fun. Switched them around!